src/functionBased/components/TodoItem.js

```diff
@@ -33,18 +36,19 @@ const TodoItem = (props) => {
     editMode.display = 'none';
   }
   useEffect(() => () => {
-    console.log('Cleaning up...');
+    window.console.log('Cleaning up...');
```

Comment:
I kindly suggest you remove the console.log as it is not necessary for any potential user.